user-guide/modules/ROOT/pages/reporting-issues.adoc

```diff
@@ -6,37 +6,24 @@
 
 == Reporting Bugs
 
-1. If you know where to look in the source code, make sure
- the bug isn't already fixed in the latest sources. The most
- recent version of everything on the Boost web site is
- available from the https://github.com/boostorg/wiki/wiki/Getting-Started:[git repositories].
-2. https://github.com/boostorg/:[Search the issues]
- on GitHub to make sure we don't already know about the bug.
- If we do, you can add further information to an existing bug ticket.
-3. https://github.com/boostorg/:[Create a new issues]
- in the repository of the particular library of your interest.
+. If you know where to look in the source code, make sure the bug isn't already fixed in the latest sources. The most  recent version of everything on the Boost web site is available from the https://github.com/boostorg/wiki/wiki/Getting-Started:[git repositories].
 
+. https://github.com/boostorg/:[Search the issues] on GitHub to make sure we don't already know about the bug. If we do, you can add further information to an existing bug ticket.
 
-If possible,
-
-	* Describe the problem carefully, including steps required to
-	 reproduce it by a library maintainers.
-	* Attach a *minimal* and *complete*
-	 program that reproduces the problem. Aside from helping
-	 the library maintainer fix the problem, you may find
-	 the bug in your own code, which can avoid a costly
-	 delay waiting for a response.
+. https://github.com/boostorg/:[Create a new issues] in the repository of the particular library of your interest.
 
 
-== If You Have a Fix for the Bug
+If possible,
 
-After completing the steps above, please submit a patch
- or a pull request as described in
- https://svn.boost.org/trac/boost/wiki/StartModPatchAndPullReq:[the wiki "Getting Started with Patches and Pull Requests"].
+[disc]
+* Describe the problem carefully, including steps required to reproduce it by a library maintainers.
 
+* Attach a *minimal* and *complete* program that reproduces the problem. Aside from helping the library maintainer fix the problem, you may find the bug in your own code, which can avoid a costly delay waiting for a response.
 
 
+== If You Have a Fix for the Bug
 
+After completing the steps above, please submit a patch or a pull request as described in https://svn.boost.org/trac/boost/wiki/StartModPatchAndPullReq:[the wiki "Getting Started with Patches and Pull Requests"].
 
 Copyright Aleksey Gurtovoy(/users/people/aleksey_gurtovoy.html) 2002
```

Comment:
This link is not rendered properly. I think I would also put this in a box, admonition, or quotation because it looks weird as part of the content.

Reply:
OK